lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -64,6 +64,11 @@ class @StudentAdmin
     @$request_response_error_ee       = @$section.find ".entrance-exam-grade-container .request-response-error"
     @$request_response_error_all    = @$section.find ".course-specific-container .request-response-error"
 
+    $student_grade_container = find_and_assert @$section, ".student-grade-container"
```

Comment:
@dan-f As i mentioned above we are populating username and problem_id in instructor dashboard and scrolling user down to that section when user click on link in ```staff_debug_modal```.